README.md

```diff
@@ -63,6 +63,7 @@ Issues can be funded by anyone and the money will be transparently distributed t
 
 * [`Primitive`](#primitive)
 * [`Falsey`](#falsey)
+* [`mixed`](#mixed)
```

Comment:
Could you please move it as the first item in "Flow's Utility Types" section?

Reply:
Does it count as utility type tho?

Reply:
It's only for Flow's migration, nobody using primarily TS world would normally want to use it